spdmlib/src/requester/psk_exchange_req.rs

```diff
@@ -140,7 +140,7 @@ impl<'a> RequesterContext<'a> {
                         // create session - generate the handshake secret (including finished_key)
                         let th1 = self
                             .common
-                            .calc_req_transcript_hash(true, &message_k, None)?;
+                            .calc_req_transcript_hash(0, true, &message_k, None)?;
```

Comment:
why hardcode 0?
I recommend use INVALID_SLOT, to indicate this is PSK.

Reply:
> why hardcode 0? I recommend use INVALID_SLOT, to indicate this is PSK.

Updated! please check the latest commit.